main.go

```diff
@@ -85,10 +95,15 @@ func main() {
 	}
 	viper.Set(types.AllowedAnnotationsMap, allowedAnnotations)
 
-	shLog := ctrl.Log.WithName("servicehandler").WithName("gcloudServiceDirectoryHandler")
+	// Create the service registry handler
```

Comment:
Maybe we should clarify that we are creating a service registry handler for GCP Service Directory?

Reply:
Just amended. Thank you!